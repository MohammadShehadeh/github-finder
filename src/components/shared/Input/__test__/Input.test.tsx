import renderer from 'react-test-renderer';

import { Input } from '..';

describe('Input', () => {
  it('should render to match snapshot', () => {
    const tree = renderer.create(<Input />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
