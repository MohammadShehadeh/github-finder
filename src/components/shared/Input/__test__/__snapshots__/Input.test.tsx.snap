// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render to match snapshot 1`] = `
<input
  className="input"
/>
`;
